### ISSUE TITLE
stm.enrich_from_polygon cannot handle polygons with holes

### ISSUE THREAD

**Issue by Simon-van-Diepen** [MotionbyLearning/stmtools#31]
If a polygon in a geopackage has holes, the query only works for a small subset of the points that are in that polygon. Other polygons are unaffected.

**Comment by rogerkuou** [comment]
Hi Simon, thanks for submitting this issue! Could you please attach a small example here to help me reproduce this issue? Ideally it can be:

- a code example, and
- a subset data of your points and polygons

I will take a look this week.

Thanks in advance!

**Comment by Simon-van-Diepen** [comment]
[AoI_EUPs_base.gpkg.zip](https://github.com/MotionbyLearning/stmtools/files/12279003/AoI_EUPs_base.gpkg.zip)

In this geopackage entries 44, 45, 47, 99, 101, and 111 have holes. All lie in the region of AHN tiles 31DN2 and 31BZ2.

**Comment by Simon-van-Diepen** [comment]
@rogerkuou yes let's have a look at this together, looking at the notebook it indeed seems to work but that wouldn't explain my issue

**Comment by rogerkuou** [comment]
After checking with Simon, we agreed that for now we need not reproduce this. For now I will close this issue.
